modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/index.es.js

```diff
@@ -38,6 +38,8 @@ export {default as postForm} from './liferay/util/form/post_form.es';
 // Liferay API
 
 export {default as BREAKPOINTS} from './liferay/breakpoints';
+export {default as STATUS_CODE} from './liferay/status_code';
+export {default as zIndex} from './liferay/zIndex';
```

Comment:
Should this be here? We don't touch anything else with zIndex here in this PR

Reply:
@bryceosterhaus I coupled these changes with this PR when Diego commented and reminded me that we should be able to import these instead of using the global namespace

Reply:
It's basically to also give the ability to import `zIndex`, because initially I just added it to the global exports. So I proposed to add it in this PR instead of opening a new one. @bryceosterhaus